createFeatureFlag/main.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"encoding/json"
+	"github.com/Real-Dev-Squad/feature-flag-backend/database"
+	"github.com/Real-Dev-Squad/feature-flag-backend/models"
+	"github.com/Real-Dev-Squad/feature-flag-backend/utils"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/go-playground/validator/v10"
+	"github.com/google/uuid"
+	"log"
+	"net/http"
+	"time"
+)
+
+var validate *validator.Validate
+
+func init() {
+	validate = validator.New()
+}
+
+func createFeatureFlag(db *dynamodb.DynamoDB, createFeatureFlagRequest utils.CreateFeatureFlagRequest) error {
+	featureFlag := models.FeatureFlag{
+		Id:          uuid.New().String(),
+		Name:        createFeatureFlagRequest.FlagName,
+		Description: createFeatureFlagRequest.Description,
+		CreatedBy:   createFeatureFlagRequest.UserId,
+		UpdatedBy:   createFeatureFlagRequest.UserId,
+		CreatedAt:   time.Now().Unix(),
+		UpdatedAt:   time.Now().Unix(),
+		Status:      utils.ENABLED,
+	}
+
+	//marshal to ddb attribute value
```

Comment:
code is explanatory, do we need this comment?

Reply:
okay

Reply:
removed